types/linkifyjs/react.d.ts

```diff
@@ -0,0 +1,12 @@
+import * as React from "react";
+import { LinkifyOptions } from "./index";
+
+interface LinkifyProps {
+    options: LinkifyOptions;
+}
+
+declare class Linkify extends React.Component<LinkifyProps> {}
+
+export { LinkifyProps };
```

Comment:
I think you can just do 

```ts
export interface LinkifyProps {
```

above but I'm not sure -- let me know if that works!

Reply:
👍